app/controllers/planning_application/notes_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class PlanningApplication
+  class NotesController < ApplicationController
+    before_action :set_planning_application
+
+    def index
+      @notes = @planning_application.notes
+      @note = Note.new
+
+      respond_to do |format|
+        format.html
+      end
```

Comment:
we can get rid of this one given there is only one format of response

Reply:
It's better to always be explicit for what formats we will accept
https://blog.arkency.com/2016/07/respond-to-format-is-useful-even-without-multiple-formats/

Reply:
rails docs say the action without respond_to implicitly responds to all formats - 
https://apidock.com/rails/ActionController/MimeResponds/respond_to - there was also an old respond_to method that is now deprecated and may have been what that blog was about.

I only suggested it because a senior suggested the same to me in the past, saying it was unnecessary - but its all the same to me if you want to keep it 🤷